### PR TITLE
feat(fibonacciSum): finding the sum of fibonacci series

### PR DESCRIPTION
**Pull request by ravikishorethella**
I've added a fibonacciSum function which return the sum of the fib series for the given number.

#132

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,2 @@
+{
```
Comment (kentcdodds) [inline .vscode/settings.json]: Could you remove this file? You may consider adding `.vscode` to your `.gitignore_global` file.

Comment (ravikishorethella) [reply .vscode/settings.json]: @kentcdodds Please check it now!

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,5 @@ node_modules
 .nyc_output/
 coverage/
 npm-debug.log
-package-lock.json
\ No newline at end of file
+package-lock.json
+.vscode
```
Comment (kentcdodds) [inline .gitignore]: Don't forget to add `.vscode` to [your global gitignore file](https://help.github.com/articles/ignoring-files/) as well.

File: .vscode/settings.json (hunk 1 above)
Comment (kentcdodds) [reply .vscode/settings.json]: Please remove this file as well:

```
git rm -rf .vscode
```

File: .gitignore (hunk 2 above)
Comment (ravikishorethella) [reply .gitignore]: Will do!

**Comment by ravikishorethella** [comment]
@kentcdodds Thanks for the guidance :+1:

**Pull request merged**